pkg/pattern/aggregation/push_test.go

```diff
@@ -144,37 +144,37 @@ func Test_Push(t *testing.T) {
 
 		p.WriteEntry(
 			wayBack,
-			AggregatedMetricEntry(model.TimeFromUnix(wayBack.Unix()), 1, 1, "test_service", lbls1),
+			AggregatedMetricEntry(model.TimeFromUnix(wayBack.Unix()), 1, 1, lbls1),
```

Comment:
If this is re-usable between aggregated metrics and patterns, maybe a better name would be "MetricEntry"?